### ISSUE TITLE
Travis CI: run multisite/non-multisite PHPUnit tests together

### ISSUE THREAD

**Issue by JDGrimes** [WordPoints/wordpoints#240]
We can cut the number of builds in half by running our multisite and non-multisite tests together, instead of as separate build environments. I'm not sure why this was originally designed this way in the wp-dev-lib, but possibly it is related to the tests being installed differently prior to xwp/wp-dev-lib#34. Anyway, I believe that we can run them both without issue. The installation is not affected by the `WP_MULTISITE` environment variable. Perhaps a prior concern was that things might not be totally clean if we were running the tests multiple times on a single install. However, they should leave the database in pristine condition, and in fact we already run the network wide and regular multisite tests together on a single build.

In short, I don't foresee any issues, and it could speed up our build time significantly by halving the time spent installing WordPress.

**Comment by JDGrimes** [comment]
This cuts our build time from 49 to 17 minutes! Woohoo!
